packages/server/build.gradle

```diff
@@ -33,7 +33,10 @@ apply plugin: 'com.diffplug.spotless'
 
 group = 'software.uncharted'
 version = '1.0.0-SNAPSHOT'
-sourceCompatibility = '17'
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
We don't seem to need this anymore - running with Java 22 and removed this and the server seems to be running fine